client/src/featureform/resources.py

```diff
@@ -603,6 +607,10 @@ def _create_training_set_resource(self, db) -> None:
 
     def _insert_training_set_features(self, db) -> None:
         for feature in self.features:
+            try:
```

Comment:
I see the features are here. Can you move the label call inside of this functions instead and change the name to insert_training_set_resources ? 

If it makes sense

Reply:
Yeah its just a little odd that the label one uses the database directly, but the training set one has its own function that takes the datbase as an arg. Feel like both of them should just have a function or be in the same function

Reply:
Okayy, Ill modify the function and put them together